code-challenges.js

```diff
@@ -8,54 +8,185 @@
 
 // Pseudo code:
 
+//problem: given a variable with a certain temperature, create a function to return whether the provided variable (temperature) is below, above or at boiling point. Boiling point is 212 degrees Fahrenheit.
+//function expression: if a temperature is less than 212 then the return should be "<temp> is below boiling point". if a temperature is greater than 212 the return should be "<temp> is above boiling point". if a temperature is equal to 212 then the return should be "212 is at boiling point."
+//set parameters: 
+    //boiling point num === 212
+    //temp < 212 return "<temp> is below boiling point"
+    //temp > 212 return "<temp> is above boiling point"
+    //temp = 212 return "<temp> is at boiling point"
+//make a decision: use if and else if statements with <.>.= when making the function for the three possible returns.
+//return should be one of the three string interpolations of below, above or at boiling point with the temperature listed. 
+
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+if(temp1 < 212) {
+    console.log(`${temp1} is below boiling point`)
+}if(temp2 > 212) {
+    console.log(`${temp2} is above boiling point`)
+}if(temp3 === 212) {
+    console.log(`${temp3} is at boiling point`)
+}
+
+//Output: 42 is below boiling point
+//350 is above boiling point
+//212 is at boiling point
+
+// I had to keep swtiching the conditional statements but I eventually got the ouput that I wanted. I originally had if and else if statements because I was following the example in the syllabus. I'm still trying to understand the differences between them and how and when to use them. I can break down the function in words but putting it together in code has been a struggle for me since jumpstart. 
+
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 // Pseudo code:
 
+//problem: combining two arrays and return how many elements are in the new combined array.
+//function expression: use .length on new array to return the number of elements in the new array
+//set parameters: both arrays need to be added together in order to get the total number of elements.
+//make a decision: use the addition symbol to add the length of each array. Use ".length" after each variable to get the length of each array. 
+//return: the total number of elements of merged arrays console.log(myNumbers1.length + myNumbers2.length)
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+console.log(myNumbers1.length + myNumbers2.length)
+
+//Output: 10
+
+//I honestly did not think that would work. I've told myself to stop over thinking and try the simplest way to work the problem first then start branching out.
```

Comment:
Yeah, it can be very easy to overthink some of these.  Always recommend breaking them down into small pieces and see what you need vs what you don't.